models/wholeBrainSeg_Large_UNEST_segmentation/docs/license.txt

```diff
@@ -0,0 +1,4 @@
+Third Party Licenses
+-----------------------------------------------------------------------
+
+/*********************************************************************/
```

Comment:
Do you have any license information for the dataset used in this PR? You can refer to:
https://github.com/Project-MONAI/model-zoo/blob/dev/models/spleen_ct_segmentation/docs/license.txt#L5

Thanks.

Reply:
No, the dataset for interence and training are public, but the labels are partly internal. License is good here. Thanks for confirming.

Reply:
I think even for the public dataset, it may have the license? like Decathlon dataset.
Could you please help confirm it?

Thanks.